lib/ui/painting.dart

```diff
@@ -1297,6 +1297,29 @@ class Canvas extends NativeFieldWrapperClass2 {
                    List<dynamic> paintObjects,
                    ByteData paintData) native "Canvas_drawCircle";
 
+  /// Draw an arc scaled to fit inside the given rectangle. It starts from
+  /// startAngle radians around the oval up to startAngle + sweepAngle
+  /// radians around the oval, with zero radians being the point on
+  /// the right hand side of the oval that crosses the horizontal line
+  /// that intersects the center of the rectangle and with positive
+  /// angles going clockwise around the oval. If useCenter is true, the oval
+  /// center is inserted into the implied path before the arc and the path is
+  /// closed back to the center, forming a wedge. Otherwise, the implied path
+  /// contains just the arc and is not closed.
+  void drawArc(Rect rect, double startAngle, double sweepAngle, bool useCenter, Paint paint) {
```

Comment:
what happens if useCenter is false and paint says to fill? Does it close the path and fill?